emacs.fnl

```diff
@@ -117,17 +111,6 @@
                                 (: fsm :toIdle)
                                 (full-screen))))
 
-;; adds Emacs modal state to the FSM instance
-(fn add-state [modal]
```

Comment:
state machine implementation has changed, this Fn afaik no longer required. Not sure if it would work the same way anyway.

Reply:
Correct, config.fnl does have an emacs-config section to cover these features.